[MTM-63273]: Cleanup of Device Certificate Authentication Documentation (#3235)
Cumulocity-IoT/c8y-docs: content/device-integration/mqtt-bundle/mqtt-examples.md

##### To generate a valid certificate {#to-generate-a-valid-certificate}

> If you don't have a valid certificate, you can generate one for testing purposes following this tutorial [Device certificates](/device-certificate-authentication/device-certificates/), following the instructions below.

Reviewer: ```suggestion
If you don't have a valid certificate, you can generate one for testing purposes following the instructions in [Device certificates](/device-certificate-authentication/device-certificates/).
```

Reviewer: This is not really clear. Should I follow the instructions below or those in the device certificates documentation?

Author: Previously this section is under Device certificate so end user first get knowledge of creating certificates and then use it in these exaples.
